src/swarm/neo/client/mixins/Controllers.d

```diff
@@ -191,6 +191,20 @@ template Controllers ( )
 
         private Pending pending;
 
+
+        /***********************************************************************
+
+            Internal state of this ISuspendable. Note that the this indicates the
+            last action done through this suspendable, and it may not reflect
+            the actual reality (the streams may need time to suspend). This
+            implies that one can't control stream directly through the controller
```

Comment:
Yeah, the whole `Suspendable` class was always regarded as a temporary hack. (The doc above says: "In the future, we may adapt the suspendable/throttling code in ocean to handle this different case.")

I'd like to reconsider the whole problem before patching the hack.